src/resonanceReconstruction/rmatrix/test/fromENDFResolvedMLBW.test.hpp

```diff
@@ -0,0 +1,2024 @@
+      CHECK( 9665.5552533233167 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 1e-4 * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 0.16391779597504672 == Approx( xs[ elas ].value ) );
+      CHECK( 3056.6149238041858 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 1e-3 * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 0.16410333516673514 == Approx( xs[ elas ].value ) );
+      CHECK( 966.89741896145790 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 1e-2 * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 0.16597922650957414 == Approx( xs[ elas ].value ) );
+      CHECK( 306.74666951513154 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 1e-1 * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 0.18572129541119137 == Approx( xs[ elas ].value ) );
+      CHECK( 100.21064013699589 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 1. * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 0.51481436908961931 == Approx( xs[ elas ].value ) );
+      CHECK( 45.463932876992189 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 10. * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 6.0327932881822504 == Approx( xs[ elas ].value ) );
+      CHECK( 13.725382293882630 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 100. * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 7.4167045411978370 == Approx( xs[ elas ].value ) );
+      CHECK( 0.51983222202793633 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 1000. * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 6.1049261794790972 == Approx( xs[ elas ].value ) );
+      CHECK( 0.16474127115670925 == Approx( xs[ capt ].value ) );
+
+      xs = resonances( 3966.9 * electronVolt );
+      CHECK( 2 == xs.size() );
+      CHECK( 16.437892514295868 == Approx( xs[ elas ].value ) );
+      CHECK( 119.81501088384495 == Approx( xs[ capt ].value ) );
+    } // THEN
+  } // GIVEN
+} // SCENARIO
+
+std::string mlbwRh105() {
+
+  // Rh105 ENDF/B-VII LRF=1 resonance evaluation
+
+  return
+    " 4.510500+4 1.040000+2          0          0          1          04531 2151     \n"
+    " 4.510500+4 1.000000+0          0          0          1          04531 2151     \n"
+    " 1.000000-5 7.500000+0          1          2          0          04531 2151     \n"
+    " 5.000000-1 6.200000-1          0          0          1          04531 2151     \n"
+    " 1.040050+2 0.000000+0          0          0         12          24531 2151     \n"
+    "-5.000000+0 1.000000+0 1.610000+0 1.450000+0 1.600000-1 0.000000+04531 2151     \n"
+    " 5.000000+0 1.000000+0 4.900000-1 3.300000-1 1.600000-1 0.000000+04531 2151     \n"
+    "                                                                  4531 2  0     \n";
+}
+
+std::string resolvedAg107() {
+
+  // Cf252 ENDF/B-VII.0 LRF=1 resonance evaluation
+
+  return
```

Comment:
Maybe some string literals could help with avoiding the `\n`s in places like this.